contracts/test/RFQ.t.sol

```diff
@@ -19,6 +19,7 @@ contract RFQTest is StrategySharedSetup {
     using SafeMath for uint256;
     using BalanceSnapshot for BalanceSnapshot.Snapshot;
 
+    // BPS_MAX must be the same as LibConstant.BPS_MAX
```

Comment:
Yes, you're right!
But if I use it directly LibConstant.BPS_MAX, it will show `TypeError: Initial value for constant variable has to be compile-time constant.` error.

```typescript
import "contracts/utils/LibConstant.sol";
...
uint256 constant BPS_MAX = LibConstant.BPS_MAX;
```

Maybe I need to change to below:
```typescript
uint256 BPS_MAX = LibConstant.BPS_MAX;
```

Reply:
Yes, or simply replace `BPS_MAX` with `LibConstant.BPS_MAX` in the tests